src/hooks/useFetchListCallback.ts

```diff
@@ -36,7 +36,8 @@ export function useFetchListCallback(): (listUrl: string, sendDispatch?: boolean
     async (listUrl: string, sendDispatch = true) => {
       const requestId = nanoid()
       sendDispatch && dispatch(fetchTokenList.pending({ requestId, url: listUrl }))
-      return getTokenList(listUrl, ensResolver)
+      // return getTokenList(listUrl, ensResolver)
```

Comment:
Why not remove this function?

Reply:
use getTokenListV2 to fetch all whitelist token. because api has paging.  getTokenList has redundant logic (token list ,...). Nam had PR to clean it.